modules/tinymce/src/core/main/ts/api/dom/RangeUtils.ts

```diff
@@ -60,9 +62,31 @@ const RangeUtils = (dom: DOMUtils): RangeUtils => {
     );
   };
 
+  /**
+   * Returns an expanded range expanded around the entire word the provided selection was collapsed in.
```

Comment:
```suggestion
   * Returns a range expanded around the entire word the provided selection was collapsed within.
```
Suggested copy edit: removed the first _expanded_ as redundant and wordy.